Modules/_decimal/_decimal.c

```diff
@@ -3735,7 +3735,7 @@ dec_##MPDFUNC(PyObject *self, PyObject *args, PyObject *kwds)  \
 static PyObject *                                                \
 dec_##MPDFUNC(PyObject *self, PyObject *args, PyObject *kwds)    \
 {                                                                \
-    static char *kwlist[] = {"other", "context", NULL};          \
+    static const char *kwlist[] = {"other", "context", NULL};          \
```

Comment:
Few other alignment issues like the ones above - probably best to align the newline escapes as above.